dataprofiler/tests/profilers/test_base_option.py

```diff
@@ -0,0 +1,54 @@
+import os
+import unittest
+from unittest import mock
+
+import pandas as pd
+
+from dataprofiler import Data, ProfilerOptions, Profiler
+from dataprofiler.profilers.profiler_options import BaseOption
+
+
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.'
+            'DataLabelerColumn.update', return_value=None)
+@mock.patch('dataprofiler.profilers.data_labeler_column_profile.DataLabeler')
+class TestBaseOption(unittest.TestCase):
+	
+	@classmethod
+	def setUpClass(cls):
+		cls.data = Data(data=pd.DataFrame([1, 2]), data_type='csv')
+	
+	def sanity(self, *mocks):
```

Comment:
what is the intention of sanity?

Reply:
Would recommend naming it something more clear, assuming this was a "sanity check"

Reply:
agree

Reply:
I usually add these to make sure the object can be created and that it has the right property keys. I should probably rename this to test_default_config or test_init

Reply:
Changed to test_init and also added a check to make sure that the properties are empty